src/screens/ChatDetailPage/index.js

```diff
@@ -221,7 +221,7 @@ const styles = StyleSheet.create({
     marginBottom: 12
   },
   containerDate: {
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the `containerDate` has been changed from `COLORS.blackgrey` to `COLORS.gray400`. If the `gray400` color is lighter than `blackgrey`, it might affect the visibility of the date text, especially if the text color is also light. Please verify that the contrast between the background and the text color is sufficient for readability.
```diff
-    backgroundColor: COLORS.blackgrey,
+    backgroundColor: COLORS.gray400,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->